esadapter/src/main/java/com/ncc/neon/adapters/es/ElasticsearchQueryConverter.java

```diff
@@ -430,7 +431,17 @@ public static IndexRequest convertMutationInsertQuery(MutateQuery mutateQuery) {
                 .source(mutateQuery.getFieldsWithValues());
     }
 
-    public static DeleteRequest convertMutationDeleteQuery(MutateQuery mutateQuery) {
+    public static DeleteRequest convertMutationDeleteByIdQuery(MutateQuery mutateQuery) {
         return new DeleteRequest(mutateQuery.getDatabaseName(), mutateQuery.getTableName(), mutateQuery.getDataId());
+
+    }
+
+    public static DeleteByQueryRequest convertMutationDeleteByFilterQuery(MutateQuery mutateQuery) {
+        SelectClause selectClause = new SelectClause(mutateQuery.getDatabaseName(), mutateQuery.getTableName());
+        QueryBuilder queryBuilder = convertWhereClauses(selectClause, List.of(mutateQuery.getWhereClause()));
+        DeleteByQueryRequest request = new DeleteByQueryRequest();
+        request.setQuery(queryBuilder);
+        request.getSearchRequest().indices("_all");
```

Comment:
I think the `indices` should be `mutateQuery.getTableName()` instead of `_all`?  See the `createSearchRequest` method in this class for a parallel use case.

Reply:
it was `mutateQuery.getDatabaseName()`. Fixed.